pyracf/resource/resource_admin.py

```diff
@@ -589,3 +718,43 @@ def _format_profile(self, result: dict) -> None:
 
         del result["securityResult"]["resource"]["commands"][0]["messages"]
         result["securityResult"]["resource"]["commands"][0]["profiles"] = profiles
+
+    def __validate_access_level(
```

Comment:
Should this be called `__valid_access_level()` since we are potentially validating several access levels.

Reply:
sure, change made.